Trakt/Trakt.csproj

```diff
@@ -18,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.2.0" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
```

Comment:
There's 0 guarantee that 10.3 or 10.4 is compatible, so this doesn't seem right

Reply:
There's no better solution TBH. This is a complete dependency hell. IMO every plugin should always be rebuilt with each new release whether it needs it or not, but having everything in separate repos makes that very daunting unless I allow myself to push to master (not force-push, just normal push). This is the nicest workaround possible, where a rebuilt just grabs the latest NuGet version.